apps/style/code-studio/pd.scss

```diff
@@ -42,6 +42,22 @@
       margin: 0;
     }
   }
+
+  table.table.table-auto-width {
```

Comment:
I think you can still scope this further. Right now this styling would apply to everything in studio because of #application-container